test/integration/features/basic.feature

```diff
@@ -97,7 +97,7 @@ Feature: Basic test
 
     @darwin @linux @windows
     Scenario: CRC status and disk space check
-        When with up to "15" retries with wait period of "1m" command "crc status --log-level debug" output should not contain "Stopped"
+        When with up to "8" retries with wait period of "2m" command "crc status --log-level debug" output should not contain "Stopped"
```

Comment:
why a 2 minutes granularity instead of 1 minute?

Reply:
In general, I agree with @cfergeau that we should keep the frequency down as much as possible so the test can move on if it gets lucky. I think I unified all "retries" to 15x1min in PR #985.

Reply:
@jsliacan ack if you want you take care of this then I will drop the retry commit.

Reply:
@praveenkumar it seems like your PR will go in sooner, so it's probably good to keep the retry changes in here. I can then basically delete the other PR.

Reply:
@jsliacan too late you need to update your PR :)